runtime/common/src/lib.rs

```diff
@@ -339,13 +339,64 @@ parameter_types! {
 		.saturating_sub(BlockExecutionWeight::get());
 }
 
-orml_traits::parameter_type_with_key! {
+pub const ACA: CurrencyId = CurrencyId::Token(TokenSymbol::ACA);
+pub const AUSD: CurrencyId = CurrencyId::Token(TokenSymbol::AUSD);
+pub const DOT: CurrencyId = CurrencyId::Token(TokenSymbol::DOT);
+pub const LDOT: CurrencyId = CurrencyId::Token(TokenSymbol::LDOT);
+pub const KAR: CurrencyId = CurrencyId::Token(TokenSymbol::KAR);
+pub const KUSD: CurrencyId = CurrencyId::Token(TokenSymbol::KUSD);
+pub const KSM: CurrencyId = CurrencyId::Token(TokenSymbol::KSM);
+pub const LKSM: CurrencyId = CurrencyId::Token(TokenSymbol::LKSM);
+pub const XBTC: CurrencyId = CurrencyId::Token(TokenSymbol::XBTC);
+pub const RENBTC: CurrencyId = CurrencyId::Token(TokenSymbol::RENBTC);
+pub const POLKABTC: CurrencyId = CurrencyId::Token(TokenSymbol::POLKABTC);
+pub const PLM: CurrencyId = CurrencyId::Token(TokenSymbol::PLM);
+pub const PHA: CurrencyId = CurrencyId::Token(TokenSymbol::PHA);
+pub const SDN: CurrencyId = CurrencyId::Token(TokenSymbol::SDN);
```

Comment:
I think we can generate those in with macro

https://github.com/AcalaNetwork/Acala/blob/master/primitives/src/currency.rs#L13